modules/ngx_http_upstream_dyups_module/ngx_http_dyups_module.c

```diff
@@ -2125,6 +2148,141 @@ ngx_http_dyups_read_msg_locked(ngx_event_t *ev)
 }
 
 
+static void ngx_dyups_restore_upstreams_from_hist_queue()
```

Comment:
hist means `history`?

Reply:
yes, if it's improper for tengine, i can adjust it.